web/templates/flatpages/default.html

```diff
@@ -61,8 +61,8 @@
                         <li><a href="https://accessibility.mit.edu/">{% trans "Accessibility" %}</a></li>
 						<li><a href="/contact_us/">{% trans "Contact us" %}</a></li>
 						<li>{% trans "Connect" %}:
-						<a href="https://www.facebook.com/lookit.mit.edu" target="_blank" rel="noopener"><i class="fa fa-facebook fa-lg social-link"></i> </a>
-						<a href="https://www.instagram.com/babiesoflookit/" target="_blank" rel="noopener"><i class="fa fa-instagram fa-lg social-link"></i> </a>
+						<a href="https://www.facebook.com/lookit.mit.edu" target="_blank"><i class="fa fa-facebook fa-lg social-link"></i> </a>
+						<a href="https://www.instagram.com/babiesoflookit/" target="_blank"><i class="fa fa-instagram fa-lg social-link"></i> </a>
```

Comment:
There's a security issue with `target="_blank"` links. `rel="noopener"` is the fix for this issue and should remain.

Reply:
done